### PR TITLE
[jobs/backup]Feat: delete Dropbox backups older than 30 days

### LINKED ISSUE #114: Run Backups in the `jobs` Container
The jobs container is used for running recurrent jobs like updating the trending pages and the spam IP list. Backup is also a recurrent job, so there is no need to make a separate container for it.

### TODO
- [x] Fix the backup script.
- [ ] Make the backup script delete backups older than one month (excluding the last backup that has images).
- [ ] Merge the backup script into the jobs container.

### PR DESCRIPTION
**Pull request by shikharish**
Closes #114.

### CODE REVIEW

File: jobs/crontab [hunk 1]
```diff
@@ -4,4 +4,5 @@
 13 3 * * 1 /root/MetaMaint.sh
 47 3 * * * /root/update_spam_denylist.sh
 02 3 * * * logrotate /root/mysql_logrotate.conf
-23 4 * * * /root/backup/run_backup.sh >> /var/log/cron.log 2>&1
\ No newline at end of file
+23 3 * * * /root/backup/run_backup.sh >> /var/log/cron.log 2>&1
+23 4 * * * /root/backup/rotate_backups.sh >> /var/log/cron.log 2>&1
```
Comment (harshkhandeparkar) [inline jobs/crontab]: Can we have separate log files for these? Also, do we rotate these logs?

File: jobs/backup/rotate_backups.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+access_token = os.environ["DROPBOX_ACCESS_TOKEN"]
+refresh_token = os.environ["DROPBOX_REFRESH_TOKEN"]
+client = dropbox.Dropbox(
+    app_key=app_key,
+    app_secret=app_secret,
+    oauth2_access_token=access_token,
+    oauth2_refresh_token=refresh_token,
+)
+
+backup_folder_path = ""
```
Comment (harshkhandeparkar) [inline jobs/backup/rotate_backups.py]: If these are constants, it would be better to name them in all caps.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: jobs/backup/rotate_backups.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    oauth2_access_token=access_token,
+    oauth2_refresh_token=refresh_token,
+)
+
+BACKUP_FOLDER_PATH = ""
+has_more_files = True
+cursor = None
+result = None
+files = []
+now = datetime.datetime.now()
```
Comment (harshkhandeparkar) [inline jobs/backup/rotate_backups.py:23]: This is also a constant right?

File: jobs/backup/rotate_backups.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    has_more_files = result.has_more
+    files.extend(result.entries)
+
+print("Starting rotation")
+for file in files:
+    if file.name.find("metakgp_wiki") == -1:
+        continue
+    file_timestamp = file.client_modified
+    days_old = (now - file_timestamp).days
+    if days_old > 30:
```
Comment (harshkhandeparkar) [inline jobs/backup/rotate_backups.py]: Should we also check for the number of files left? If the backup job fails for some reason, over time this job will delete all backup files.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Review by harshkhandeparkar — APPROVED** [review]
"Jai Shree Ram! chal jayega"
\- Shikhar

**Review by rajivharlalka — APPROVED** [review]

**Pull request merged**